bundles/org.openhab.core.model.script.runtime/src/org/openhab/core/model/script/runtime/internal/engine/ScriptEngineImpl.java

```diff
@@ -184,4 +187,11 @@ public String getExtension() {
         return "script";
     }
 
+    private void deleteResource(Resource resource) {
+        try {
+            resource.delete(Collections.emptyMap());
+        } catch(IOException e) {
+            // Do nothing
```

Comment:
In general it isn't a bad idea to at least do a debug logging in such a situation, but I'm ok without it here.